<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_only_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget {
	background-color: rgb(105, 185, 255);
}
QPushButton {
	color: white;
	height: 50px;
	border: none;
	border-radius: 10px;
}
QPushButton:checked {
	background-color: rgb(245, 250, 254);
	color: rgb(105, 185, 255);
	font-weight: bold;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::MarkdownText</enum>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/Images/kali.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="dashboard_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Images/dashboard_white.png</normaloff>
             <normalon>:/Images/dashboard.png</normalon>:/Images/dashboard_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="profile_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Images/profile_white.png</normaloff>
             <normalon>:/Images/profile.png</normalon>:/Images/profile_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="messages_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Images/messages_white.png</normaloff>
             <normalon>:/Images/messages.png</normalon>:/Images/messages_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="notifications_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Images/notifications_white.png</normaloff>
             <normalon>:/Images/notifications.png</normalon>:/Images/notifications_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settings_1">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Images/settings_white.png</normaloff>
             <normalon>:/Images/settings.png</normalon>:/Images/settings_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>315</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="singout_1">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/Images/log_out_white.png</normaloff>:/Images/log_out_white.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="icon_name_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget {
	background-color: rgb(105, 185, 255);
	color: white;
}
QPushButton {
	color: white;
	text-align:left;
	height: 50px;
	font-size: 15px;
	border: none;
	padding-left: 10px;
	border-bottom-left-radius: 10px;
	border-top-left-radius: 10px;
}
QPushButton:checked {
	background-color: rgb(245, 250, 254);
	color: rgb(105, 185, 255);
	font-weight: bold;
}


</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/Images/kali.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>SideBar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="dashboard_2">
           <property name="text">
            <string>Dashboard</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Images/dashboard_white.png</normaloff>
             <normalon>:/Images/dashboard.png</normalon>:/Images/dashboard_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="profile_2">
           <property name="text">
            <string>Profile</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Images/profile_white.png</normaloff>
             <normalon>:/Images/profile.png</normalon>:/Images/profile_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="messages_2">
           <property name="text">
            <string>Messages</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Images/messages_white.png</normaloff>
             <normalon>:/Images/messages.png</normalon>:/Images/messages_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="notifications_2">
           <property name="text">
            <string>Notifications</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Images/notifications_white.png</normaloff>
             <normalon>:/Images/notifications.png</normalon>:/Images/notifications_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settings_2">
           <property name="text">
            <string>Settings</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/Images/settings_white.png</normaloff>
             <normalon>:/Images/settings.png</normalon>:/Images/settings_white.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>313</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="singout_2">
         <property name="text">
          <string>Sign out</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/Images/log_out_white.png</normaloff>:/Images/log_out_white.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="main_menu" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="menu">
            <property name="styleSheet">
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/Images/menu.png</normaloff>:/Images/menu.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>275</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_11">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/Images/search.png</normaloff>:/Images/search.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>275</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_10">
            <property name="styleSheet">
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/Images/image.png</normaloff>:/Images/image.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <widget class="QWidget" name="Dashboard_page">
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>360</x>
             <y>50</y>
             <width>211</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Malgun Gothic</family>
             <pointsize>30</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Dashboard</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Messages_page">
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>60</y>
             <width>211</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Malgun Gothic</family>
             <pointsize>30</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Messages</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Notifications_page">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>60</y>
             <width>271</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Malgun Gothic</family>
             <pointsize>30</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Notifications</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Settings_page">
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>50</y>
             <width>211</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Malgun Gothic</family>
             <pointsize>30</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Profile_page">
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>380</x>
             <y>30</y>
             <width>211</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Malgun Gothic</family>
             <pointsize>30</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Profile</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_only_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_name_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settings_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notifications_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>notifications_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notifications_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>notifications_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messages_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>messages_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messages_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>messages_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profile_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>profile_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profile_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>profile_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dashboard_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>dashboard_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singout_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>753</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>857</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singout_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>757</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>881</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singout_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>singout_1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>755</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>755</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singout_1</sender>
   <signal>toggled(bool)</signal>
   <receiver>singout_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>744</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>743</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
